### PR TITLE
fix transforms

### PR DESCRIPTION
**Pull request by mazabou**
- Incorporated `ToTensor` in the `__getitem__`
- Added deprecation error when using `mask_transform`. If `mask_transform=ToTensor()`, it is ignored, anything else raises an error. This avoid breaking any of the previous notebooks.
- Added deprecation warning when using `image_transform`, `ToTensor()` is no longer needed, and when used it only augments the image (so only useful for task 1)
- Added a new `transform` argument. This callable will be applied to both image and segmentation mask. 
- Added an example notebook on how to use transforms in task 1 and task 2. Currently only includes torchvision examples

### CODE REVIEW

**Review by erikjohnson24 — APPROVED** [review]
This looks good- it appears to address the problem and maintain backward compatibility, thanks!
We should get reviews from others- someone (maybe Matthew) should test with student note books before merge
We should discuss with eva before any merge to minimize disruptions to ongoing users.

**Comment by lakshmi-sathi** [comment]
Summarizing changes in latest commit to `transforms` branch: 
* MTNeuro was being used through the cloned repo rather than through the MTNeuro library installation \*FIXED\*
* MTNeuro library **was not able to access `UNet` model  as it was not included in `__init__.py`** \*FIXED\*
* Modified getting started notebooks to remove the use of transforms in them.
* Added some minimum details to the different cells in getting started and augmentation notebooks
* Added mention of MTNeuro and citation to getting started and augmentation notebooks
* Added `Open in Colab` button to getting started and augmentation notebooks
* tested `task1_getting_started`, `task2_getting_started`, `task3_getting_started` & `applying_augmentations` 

Note:
1) *The MTNeuro library installation and the 'Open in Colab' button will only start working once the PR is merged, due to hardcoded access of the repo main branch*

2) *Only the notebooks and their use of MTNeuro library has been tested. Original 'task1_main.py', 'task2_main.py' scripts have not been tested/re-tested*

**Review by lakshmi-sathi — APPROVED** [review]
Fixed issue with UNet model not being imported in \_\_init\_\_.py
Modified getting started notebooks to remove use of transforms
Added minimum necessary explanation lines, mention of MTNeuro and citation
Tested getting started and augmentation notebooks
Looks good to merge

**Pull request merged**
